common/util/src/test/java/org/thingsboard/common/util/JacksonUtilTest.java

```diff
@@ -32,4 +37,23 @@ public void allow_unquoted_field_mapper_test() {
         Assert.assertThrows(IllegalArgumentException.class, () -> JacksonUtil.toJsonNode(data)); // syntax exception due to missing quotes in the field name!
     }
 
-}
\ No newline at end of file
+    @Test
+    public void fail_on_unknown_properties_edge_mapper_test() throws JsonProcessingException {
```

Comment:
naming convention - please rename it according to default naming convention.
and please do the same for the test above allow_unquoted_field_mapper_test